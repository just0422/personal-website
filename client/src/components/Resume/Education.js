import React, { Component } from 'react';
import { Row, Col } from 'react-bootstrap';

import './resume_components.css';

export default class Education extends Component {
	render() {
		return (
			<div>
				<h3 className="resume-subheader">Education</h3>
				<div className="section-element-header">
					<strong>Stony Brook University, New York</strong>
				</div>
				<Row>
					<Col xs={6} className="section-element-details">M.S. Computer Science</Col>
					<Col xs={6} className="section-element-dates">Aug. 2016 - Aug. 2018</Col>
				</Row>
				<Row>
					<Col xs={6} className="section-element-details">B.S. Computer Science</Col>
					<Col xs={6} className="section-element-dates">Aug. 2012 - May 2016</Col>
				</Row>
			</div>
		);
	}
}
