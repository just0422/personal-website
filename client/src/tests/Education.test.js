import React from 'react';
import {shallow} from 'enzyme';

import Education from '../components/Resume/Education';

describe('Education', () => {
  it('should render correctly with no props', () => {
    const component = shallow(<Education />);
    expect(component).toMatchSnapshot();
  });
});
describe('Education', () => {
  it('should render correctly even with a prop', () => {
    const string = 'string';
    const component = shallow(<Education something={string} />);
    expect(component).toMatchSnapshot();
  });
});
