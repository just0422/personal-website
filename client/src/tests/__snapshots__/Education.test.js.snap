// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Education should render correctly even with a prop 1`] = `ShallowWrapper {}`;

exports[`Education should render correctly with no props 1`] = `ShallowWrapper {}`;
